libs/sdk-common/src/input_parser.rs

```diff
@@ -469,6 +472,9 @@ fn ln_address_decode(ln_address: &str) -> Result<(String, String, String)> {
         // It is safe to downcase the domains since they are case-insensitive.
         // https://www.rfc-editor.org/rfc/rfc3986#section-3.2.2
         let domain = split[1].to_lowercase();
+        if !domain.contains('.') {
```

Comment:
What caused you to add this here in this PR?

Reply:
The MockRestClient was picking up a request to get LNURL info for `03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3zzz@sdfsffs` (in the node_id parsing trst). I think TLDs need a dot right?

Reply:
Technically it could be `localhost`, or other domains in the local network. So this failed before because the network call failed?

Reply:
> So this failed before because the network call failed?

Yes. Ok, it maybe makes sense to remove this then and add a mock response for this case